### DISCUSSION TITLE
Are country-oriented servers hosted by you?

### DISCUSSION THREAD

**Discussion by bseditor** [efroemling/ballistica#441] [Q&A] [answered]
@efroemling if so, I am curious to know that how u are managing all this mess.

**Comment by efroemling** [comment, marked as answer]
Yes. And very carefully haha.

A huge amount of my time the past few years has been going towards getting the whole cloud thing automated and running smoothly. It is basically a bunch of Amazon EC2 servers running in just about every part of the world where Amazon can host servers, along with the central V2 master-server managing it all and spinning EC2 servers up and down as needed.

Right now it is mainly hosting static servers "FFA US West 1", etc. and Public Parties, but as 2.0 approaches it'll be used for lots more.

Here's one of my ugly control panels I made for monitoring it all:

<img width="922" alt="Screen Shot 2022-07-02 at 11 43 26 AM" src="https://user-images.githubusercontent.com/10857610/177012767-407d1af5-de97-4001-936d-f9cd43665081.png">

**Comment by MatiasMGz** [comment]
Not that ugly after all but very interesting.
